### DISCUSSION TITLE
Allow for cheats to carry over to new parks (like it used to work)

### DISCUSSION THREAD

**Discussion by MarcelVos96** [OpenRCT2/OpenRCT2#15967] [Feature requests]
I think the fact that cheats are now saved with the park is a great change, but them being reset when you load a new park can be annoying. I frequently load up park after park for making my videos and not having to re-enable all the convenience cheats (freeze weather, no breakdowns, etc) makes it a lot easier.

Having an option to have the cheats stay or reset upon opening a different park would be very beneficial.

**Comment by ZehMatt** [comment]
You could probably achieve this with scripting. So I don't think its worth adding.

**Reply by ocalhoun6** [reply]
Yeah, sounds like very fertile ground for a plugin.

Just have a plugin that lets you select various cheats and which remembers which ones you've selected across different saves. Then just start that plugin and you'll instantly apply all the cheats you want with one click.
